centreon/doc/API/v23.10/Configuration/Notification/Schema/PartialUpdateNotificationRequest.yaml

```diff
@@ -0,0 +1,5 @@
+type: object
+required: ["is_activated"]
```

Comment:
No properties are mandatory in a PATCH action

Reply:
done